### DISCUSSION TITLE
A Question in DRF tutorial page

### DISCUSSION THREAD

**Discussion by tc-ha** [encode/django-rest-framework#8334] [Question & Answer] [answered]
at https://www.django-rest-framework.org/tutorial/1-serialization/#getting-started, there's a code snippet; below

```
cd ~
django-admin startproject tutorial
cd tutorial
```

in the second line, shouldn't it be "django-admin startproject tutorial **.**" rather then as it is now?

**Comment by lovelydinosaur** [comment, marked as answer]
I don't think so - I just tried out the opening bits, and ended up with the same project structure as https://github.com/encode/rest-framework-tutorial - looked correct to me.
